robottelo/hosts.py

```diff
@@ -479,7 +509,27 @@ def execute_foreman_scap_client(self, policy_id=None):
         if result.status != 0:
             raise ContentHostError('Failed to execute foreman_scap_client run.')
 
-    def configure_rhai_client(self, activation_key, org, rhel_distro, register=True):
+    def configure_rex(self, satellite, org, subnet_id=None, by_ip=True):
+        """Setup a VM host for remote execution.
+
+        :param Satellite satellite: a hosts.Satellite instance
+        :param str org: The organization entity, label attr is used
+        :param int subnet: (Optional) Nailgun subnet entity id, to be used by the vm_client host.
+        :param bool by_ip: Whether remote execution will use ip or host name to access server.
+        """
+        self.install_katello_ca(satellite)
+        self.register_contenthost(org.label, lce='Library')
+        assert self.subscribed
```

Comment:
This should be optional, allowing users to configure rex on already registered hosts.

Reply:
ack, going to default to doing these steps though since all uses of it currently expect it. Pass bool false for a kwarg to disable it on demand.